inference-engine/samples/speech_sample/main.cpp

```diff
@@ -708,10 +737,28 @@ int main(int argc, char *argv[]) {
 
         /** Stores all input blobs data **/
         std::vector<Blob::Ptr> ptrInputBlobs;
-        for (auto& input : cInputInfo) {
-            ptrInputBlobs.push_back(inferRequests.begin()->inferRequest.GetBlob(input.first));
+        if (!FLAGS_iname.empty()) {
```

Comment:
why input blobs might not be initialized? Will the be set to 0? What is use case? What are scale factors would be - they need to be provided explicitly then - right for every missed input?